assets/js/hooks/user_form.js

```diff
@@ -11,8 +11,9 @@ const UserForm = {
   mounted() {
     this.el.addEventListener("submit", (event) => {
       const name = this.el.user_form_name.value;
+      const email = this.el.user_form_email.value;
```

Comment:
We should not have email in the form, since we can verify it. In this case, the email will always be `nil`.